Libraries/Text/TextInput/RCTInputAccessoryShadowView.m

```diff
@@ -14,7 +14,9 @@ @implementation RCTInputAccessoryShadowView
 - (void)insertReactSubview:(RCTShadowView *)subview atIndex:(NSInteger)atIndex
```

Comment:
InputAccessoryView is used by several RNTester tests.   Like Android, the macOS version of this component will show the YellowBox warning "not available on this platform" if App code tries to actually call it.   But due to the way the shared JS code is shaped, it requires a NativeModule for all platforms.   It was trivial to get the iOS code building for macOS.